.github/workflows/semgrep-ci.yml

```diff
@@ -14,13 +14,15 @@ on:
 
 jobs:
   semgrep:
+    name: Semgrep Code Quality Scan
     runs-on: ubuntu-latest
+    container:
+      image: returntocorp/semgrep-action
```

Comment:
Does this actually work? The PR you raised is not from a fork, the action will use latest semgrep anyway AFAIK. 👀

Reply:
@ankush Thanks for spotting that. Image should be `returntocorp/semgrep`, not `returntocorp/semgrep-action`.

Reply:
https://github.com/hashicorp/terraform-provider-aws/pull/25508